README.md

```diff
@@ -35,6 +35,8 @@ While the following requirements are meant to be for production, the charm can b
 - 12 storage devices
 - 10 GbE card
 
+The charm is meant to be deployed using juju 2.9.38 using jammy as a series. 
```

Comment:
**suggestion:** Would change wording to something like:

Requires `juju>=2.9.37`

Jammy as a series isn't needed as it's inferred by that version.

Reply:
but one could request that. I mean: I believe at a certain point I was specifying series in deployment (possibly when using serverstack)

Reply:
IIRC, you shouldn't need to specify `--series` if your client and agent version are above 2.9.37.
It's possible that `--series` might help but I would avoid adding that alongside the phrase: `is **meant** to be deployed using...`

Reply:
fair enough. it is not a bit point anyway